pkgs/tools/networking/dd-agent/5.nix

```diff
@@ -1,110 +0,0 @@
-{ lib, stdenv, fetchFromGitHub, python2
-, unzip, makeWrapper }:
-let
-  python' = python2.override {
-    packageOverrides = self: super: {
-      docker = self.buildPythonPackage rec {
-        name = "docker-${version}";
-        version = "1.10.6";
-
-        src = fetchFromGitHub {
-          owner = "docker";
-          repo = "docker-py";
-          rev = version;
-          sha256 = "1awzpbrkh4fympqzddz5i3ml81b7f0i0nwkvbpmyxjjfqx6l0m4m";
-        };
-
-        propagatedBuildInputs = with self; [
-          six
-          requests
-          websocket-client
-          docker_pycreds
-          uptime
-        ] ++ lib.optionals (self.pythonOlder "3.7") [ backports_ssl_match_hostname ];
-
-        # due to flake8
-        doCheck = false;
-      };
-
-      pymongo = super.pymongo.overridePythonAttrs (oldAttrs: rec {
-        version = "2.9.5";
-        src = oldAttrs.src.override {
-          inherit version;
-          sha256 = "912516ac6a355d7624374a38337b8587afe3eb535c0a5456b3bd12df637a6e70";
-        };
-      });
-    };
-  };
-
-in stdenv.mkDerivation rec {
-  version = "5.11.2";
-  pname = "dd-agent";
-
-  src = fetchFromGitHub {
-    owner  = "datadog";
-    repo   = "dd-agent";
-    rev    = version;
-    sha256 = "1iqxvgpsqibqw3vk79158l2pnb6y4pjhjp2d6724lm5rpz4825lx";
-  };
-
-  patches = [ ./40103-iostat-fix.patch ];
```

Comment:
Please remove this .patch as it's no longer needed.

Reply:
thanks!